src/set_theory/ordinal/topology.lean

```diff
@@ -234,7 +234,7 @@ begin
   { let g : ι → ordinal.{u} := λ i, (enum_ord_order_iso hs).symm ⟨_, hf i⟩,
     suffices : enum_ord s (sup.{u u} g) = sup.{u u} f,
     { rw ←this, exact enum_ord_mem hs _ },
-    rw is_normal.sup.{u u u} h g hι,
```

Comment:
I feel like you should be using a `resetI` instead of this `@`.

Reply:
Adding `resetI,` before this `rw` still doesn't work if I don't use `@`. Not sure why.